backend/modules/parsers/multimodalparser.py

```diff
@@ -182,7 +183,9 @@ async def get_chunks(
 
                     pages[page_number] = image_base64
                 except Exception as e:
-                    logger.error(f"Error in page: {page_number} - {e}")
+                    exc_info = sys.exc_info()
+                    logger.error("".join(traceback.format_exception(*exc_info)))
+                    logger.error(f"Error in page: {page_number}")
```

Comment:
This is just replicating what `logger.exception` does
Might as well use that directly

Reply:
Got it, I actually wanted to have this in your serialisation comment.

Reply:
This code is unrelated to that:
https://github.com/truefoundry/cognita/pull/258#discussion_r1660457670

Reply:
Ohh, Then how to solve the serialization?